ch_backup/zookeeper/zookeeper.py

```diff
@@ -106,7 +106,7 @@ def delete_replica_metadata(
                 table_macros.update(macros)
                 path = os.path.join(
                     self._zk_root_path,
-                    table_path[1:].format(**table_macros),
```

Comment:
`format` doesn't cover the case when ZK path contains unbalanced `{`, `}`. 

It could lead to restore failures in corner cases.
```
  File "/opt/yandex/ch-backup/lib/python3.6/site-packages/ch_backup/zookeeper/zookeeper.py", line 109, in delete_replica_metadata
    table_path[1:].format(**table_macros),
ValueError: Single '}' encountered in format string
```

```
SELECT
    database,
    table,
    zookeeper_path
FROM system.replicas
WHERE zookeeper_path LIKE '%}%'

┌─database─┬─table───┬─zookeeper_path─────────────┐
│ db1      │ table1  │ /clickhouse/tables}/table1 │
```